diff --git a/node_modules/antd/lib/input/TextArea.d.ts b/node_modules/antd/lib/input/TextArea.d.ts
index 3a098a0..aced54a 100644
--- a/node_modules/antd/lib/input/TextArea.d.ts
+++ b/node_modules/antd/lib/input/TextArea.d.ts
@@ -9,6 +9,7 @@ export interface TextAreaProps extends Omit<RcTextAreaProps, 'suffix'> {
     /** @deprecated Use `variant` instead */
     bordered?: boolean;
     size?: SizeType;
+    value?: string | null;
     status?: InputStatus;
     rootClassName?: string;
     /**
